ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -142,6 +244,90 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-heterogeneous
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec
+- as: azure-ipi-usertags-sa-encryption-arm-f28
+  cron: 1 16 4 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-ipi-usertags-storage-account-encryption
+- as: baremetal-compact-agent-ipv4-dhcp-day2-amd-mixarch-f28
+  cluster: build05
+  cron: 2 18 3 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKERS: "1"
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      DISCONNECTED: "false"
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
+      ipv4_enabled: "true"
+      ipv6_enabled: "false"
+      masters: "3"
+      workers: "0"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: baremetal-lab-agent-install-network-dynamic-day2
+- as: baremetal-compact-agent-ipv4-dhcp-disc-day2-amd-mixarch-f28
+  cluster: build05
+  cron: 35 18 18 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKERS: "1"
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      DISCONNECTED: "true"
+      RESERVE_BOOTSTRAP: "false"
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      architecture: amd64
+      ipv4_enabled: "true"
+      ipv6_enabled: "false"
+      masters: "3"
+      workers: "0"
+    test:
+    - chain: openshift-upgrade-qe-test-disconnected
+    workflow: baremetal-lab-agent-install-network-dynamic-disconnected-day2
+- as: baremetal-upi-ovn-dual-stack-arm-f28
+  cluster: build05
+  cron: 45 14 9 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      architecture: arm64
+      masters: "3"
+      workers: "2"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: baremetal-lab-upi-dual-stack
+- as: gcp-ipi-xpn-private-filestore-csi-arm-f28
+  cron: 23 19 10 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.15
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-ipi-xpn-private-filestore-csi
+- as: vsphere-ipi-compact-etcd-encryption-f28
```

Comment:
vspehre platform does not support multi payload yet, so I think the job should added in amd ci yaml.

Reply:
Actually the job already exists in amd y-stream upgrade, need to add the same job in z-stream amd upgrade ci yaml.

Reply:
I think we should not have test configs using the multi payload on an unsupported platform, even if it can work 

/hold 
Just to have a short chat about this

Reply:
@aleskandro The vsphere one on multi is already removed.

Reply:
Thanks Liang

/unhold